listenbrainz/webserver/static/js/src/FollowButton.tsx

```diff
@@ -62,7 +63,9 @@ class FollowButton extends React.Component<
       error: false,
     };
 
-    this.APIService = new APIService(`${window.location.origin}/1`);
+    const { apiUrl } = this.props;
+
+    this.APIService = new APIService(apiUrl);
```

Comment:
```suggestion
    this.APIService = new APIService(
      props.apiUrl || `${window.location.origin}/1`
    );
```

Reply:
Just for the sake of consistency with other components.

Reply:
Ref: http://livegrep.metabrainz.org/search/livegrep?q=this.APIService%20%3D%20new%20APIService&fold_case=auto&regex=false&context=true&repo%5B%5D=metabrainz%2Flistenbrainz-server